shell/src/main/java/alluxio/cli/LogLevel.java

```diff
@@ -0,0 +1,193 @@
+import alluxio.wire.WorkerNetAddress;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Preconditions;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.apache.commons.lang.StringUtils;
+
+import javax.annotation.concurrent.NotThreadSafe;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Set and get the specify target specify log's level.
+ */
+@NotThreadSafe
+public final class LogLevel {
+
+  private static final String LOG_LEVEL = "logLevel";
+  private static final String TARGET_OPTION_NAME = "target";
+  private static final Option TATGET_OPTION =
+      Option.builder().required(false).longOpt(TARGET_OPTION_NAME).hasArg(true)
+          .desc("<master|workers|host:port>. Multi target split by ','.").build();
+  private static final String LOGNAME_OPTION_NAME = "logName";
+  private static final Option LOGNAME_OPTION =
+      Option.builder().required(true).longOpt(LOGNAME_OPTION_NAME).hasArg(true)
+          .desc("log name.").build();
+  private static final String LEVEL_OPTION_NAME = "level";
+  private static final Option LEVEL_OPTION =
+      Option.builder().required(false).longOpt(LEVEL_OPTION_NAME).hasArg(true)
+          .desc("<master|workers|host:port>.").build();
+  private static final Options OPTIONS = new Options()
+      .addOption(TATGET_OPTION)
+      .addOption(LOGNAME_OPTION)
+      .addOption(LEVEL_OPTION);
+
+  private static final String USAGE =
+      "logLevel --logName=LOGNAME [--target=<master|workers|host:port>] [--level=LEVEL]";
+
+  /**
+   * Prints the help message.
+   *
+   * @param message message before standard usage information
+   */
+  public static void printHelp(String message) {
+    System.err.println(message);
+    HelpFormatter help = new HelpFormatter();
+    help.printHelp(USAGE, OPTIONS);
+  }
+
+  /**
+   * Implements log level setting and getting.
+   *
+   * @param args list of arguments
+   * @return 0 on success, 1 on failures
+   */
+  public static int logLevel(String... args) {
+    CommandLineParser parser = new DefaultParser();
+    CommandLine cmd;
+
+    try {
+      cmd = parser.parse(OPTIONS, args, true /* stopAtNonOption */);
+    } catch (ParseException e) {
+      printHelp("Unable to parse input args: " + e.getMessage());
+      return 1;
+    }
+    Preconditions.checkNotNull(cmd, "Unable to parse input args");
+    List<TargetInfo> addrList = new ArrayList<>();
+    String logName = "";
+    String level = null;
+    String[] targets;
+
+    String argName = cmd.getOptionValue(LOGNAME_OPTION_NAME);
+    if (StringUtils.isNotBlank(argName)) {
+      logName = argName;
+    }
+
+    if (cmd.hasOption(TARGET_OPTION_NAME)) {
```

Comment:
NIT: parse the options in the same order as defined in the options list, it will look more logical.